cpp/ql/lib/change-notes/2023-09-06-as-defining-argument-off-by-one-fix.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* The `DataFlow::asDefiningArgument` predicate now takes its argument from the range starting at `1` instead of `2`. Queries that depend on the single-parameter version of `DataFlow::asDefiningArgument` should have their arguments arguments accordingly.
```

Comment:
?
```suggestion
* The `DataFlow::asDefiningArgument` predicate now takes its argument from the range starting at `1` instead of `2`. Queries that depend on the single-parameter version of `DataFlow::asDefiningArgument` should have their arguments updated accordingly.
```

Reply:
Doh. You're right :) Thank you!